builtin/logical/database/backend.go

```diff
@@ -144,7 +144,7 @@ type upgradeStatements struct {
 type upgradeCheck struct {
 	// This json tag has a typo in it, the new version does not. This
 	// necessitates this upgrade logic.
-	Statements upgradeStatements `json:"statments"`
+	Statements upgradeStatements `json:"statements"`
```

Comment:
[For others in the Vault team] This may not be required.

Reply:
Will also break things.